segments/tokenizer.py

```diff
@@ -170,9 +171,9 @@ class to build a trie data structure, which stores the possible Unicode
     In an orthography rules file, rules are given in order in regular
     expressions, e.g. this rule replaces a vowel followed by an <n>
     followed by <space> followed by a second vowel with first vowel
-    <space> <n> <space> second vowel, e.g.:
+    <space> <n> <space> second vowel, e.g.::
```

Comment:
@LinguList is the extra colon necessary (or a typo)?

Reply:
Rst! The markdown is generally corrupted in the docu. Ideally to be changed in all places. Maybe check with one of the numerous Python rst guides.

Reply:
Not corrupted if it was never my intention to use RST. But your point is noted. :)